content/zh-cn/docs/concepts/cluster-administration/logging.md

```diff
@@ -112,140 +127,227 @@ See the [`kubectl logs` documentation](/docs/reference/generated/kubectl/kubectl
+and the runtime writes the container logs to the given location.
 -->
-默认情况下，如果容器重启，kubelet 会保留被终止的容器日志。
-如果 Pod 在工作节点被驱逐，该 Pod 中所有的容器也会被驱逐，包括容器日志。
+你可以将 kubelet 配置为自动轮转日志。
+
+如果配置轮转，kubelet 负责轮转容器日志并管理日志目录结构。
+kubelet 将此信息发送到容器运行时（使用 CRI），且该运行时将容器日志写入给定位置。
 
 <!--
-An important consideration in node-level logging is implementing log rotation,
-so that logs don't consume all available storage on the node. Kubernetes
-is not responsible for rotating logs, but rather a deployment tool
-should set up a solution to address that.
-For example, in Kubernetes clusters, deployed by the `kube-up.sh` script,
-there is a [`logrotate`](https://linux.die.net/man/8/logrotate)
-tool configured to run each hour. You can also set up a container runtime to
-rotate application's logs automatically.
+You can configure two kubelet [configuration settings](/docs/reference/config-api/kubelet-config.v1beta1/#kubelet-config-k8s-io-v1beta1-KubeletConfiguration),
+`containerLogMaxSize` and `containerLogMaxFiles`,
+using the [kubelet configuration file](/docs/tasks/administer-cluster/kubelet-config-file/).
+These settings let you configure the maximum size for each log file and the maximum number of files allowed for each container respectively.
+
+When you run [`kubectl logs`](/docs/reference/generated/kubectl/kubectl-commands#logs) as in
+the basic logging example, the kubelet on the node handles the request and
+reads directly from the log file. The kubelet returns the content of the log file.
 -->
-节点级日志记录中，需要重点考虑实现日志的轮转，以此来保证日志不会消耗节点上全部可用空间。
-Kubernetes 并不负责轮转日志，而是通过部署工具建立一个解决问题的方案。
-例如，在用 `kube-up.sh` 部署的 Kubernetes 集群中，存在一个
-[`logrotate`](https://linux.die.net/man/8/logrotate)，每小时运行一次。
-你也可以设置容器运行时来自动地轮转应用日志。
+你可以使用 [kubelet 配置文件](/zh-cn/docs/tasks/administer-cluster/kubelet-config-file/)配置两个
+kubelet [配置设置](/zh-cn/docs/reference/config-api/kubelet-config.v1beta1/#kubelet-config-k8s-io-v1beta1-KubeletConfiguration)、
+`containerLogMaxSize` 和 `containerLogMaxFiles`。
+这些设置允许你分别配置每个日志文件大小的最大值和每个容器允许的最大文件数。
 
+当类似于基本日志示例一样运行 [`kubectl logs`](/docs/reference/generated/kubectl/kubectl-commands#logs) 时，
+该节点上的 kubelet 会处理请求并直接从日志文件中读取。kubelet 将返回该日志文件的内容。
+
+{{< note >}}
 <!--
-As an example, you can find detailed information about how `kube-up.sh` sets
-up logging for COS image on GCP in the corresponding
-[`configure-helper` script](https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/gci/configure-helper.sh).
+Only the contents of the latest log file are available through
+`kubectl logs`.
+
+For example, if a Pod writes 40 MiB of logs and the kubelet rotates logs
+after 10 MiB, running `kubectl logs` returns at most 10MiB of data.
 -->
-例如，你可以找到关于 `kube-up.sh` 为 GCP 环境的 COS 镜像设置日志的详细信息，
-脚本为
-[`configure-helper` 脚本](https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/gci/configure-helper.sh)。
+只有最新的日志文件的内容可以通过 `kubectl logs` 获得。
+
+例如，如果 Pod 写入 40 MiB 的日志，并且 kubelet 在 10 MiB 之后轮转日志，
+则运行 `kubectl logs` 将最多返回 10MiB 的数据。
```

Comment:
Though English have no space here, we can fix it in Mandarin version.
In standard, we have to add a space between number and unit.
```suggestion
则运行 `kubectl logs` 将最多返回 10 MiB 的数据。
```